api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -22,7 +22,7 @@
  * 
  * @since 1.12
  */
-public class OrderGroup extends BaseChangeableOpenmrsData {
+public class OrderGroup extends BaseCustomizableMetadata<OrderGroupAttribute> {
```

Comment:
```suggestion
public class OrderGroup extends BaseCustomizableData<OrderGroupAttribute> {
```

The change from a subtype of `OpenmrsData` -> `OpenmrsMetadata` is causing the error.

Reply:
Oh thanks alot @ibacher  let me try to alter this. But just a quick one out of curiosity .  isn't OrderGroupAttribute supposed to OpenmrsMetaData ?

Reply:
The attributes might represent metadata, but OrderGroups themselves represent a group of actual orders placed, so they clearly belong on the "data" side of things (OrderSets, which represent a template for an OrderGroup are definitely metadata).

Reply:
Oh thanks for the clarification .It actually cleared me hours of scratching the skull.

Reply:
@tendomart if you look at the ticket description again, it is the fourth commit.

Reply:
Ohh yeah... I think being i confused it with some other Base class .Thanks @dkayiwa